src/OnboardingSPA/steps/TheFork/contents.js

```diff
@@ -10,21 +10,21 @@ const getContents = () => {
 		),
 		options: [
 			{
-				title: __( 'Guided Configuration', 'wp-module-onboarding' ),
+				title: __( ' Website Creator', 'wp-module-onboarding' ),
 				subtitle: __(
-					'Robust configuration guide to help you build your site',
+					'Unique AI generated content & design curated for you.',
```

Comment:
Should we update the `.pot` file?

Reply:
Should we do it now or with the release?